app/javascript/article-form/components/Tabs.jsx

```diff
@@ -10,11 +10,13 @@ export const Tabs = ({ onPreview, previewShowing }) => {
       <ul className="crayons-tabs__list">
         <li>
           <button
-            data-text='Edit'
+            data-text="Edit"
             className={`crayons-tabs__item ${
               previewShowing ? '' : 'crayons-tabs__item--current'
             }`}
-            onClick={previewShowing && onPreview}
+            onClick={(e) => {
+              previewShowing && onPreview(e);
+            }}
```

Comment:
With the debug enabled this was throwing an error and causing tests to fail, as when `previewShowing` is false, we'd effectively be setting a click handler to `false` instead of null.